packages/bot-web-ui/src/components/dashboard/tour-guide.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { storeSetting, getSetting } from '../../utils/settings';
+import { Text } from '@deriv/components';
+import { getUrlBase } from '@deriv/shared';
+import RootStore from 'Stores/index';
+import { connect } from 'Stores/connect';
+
+type TTourGuide = {
+    Label: string | boolean;
```

Comment:
shouldn't it start with small case? `label`

Reply:
done ✅